src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -84,26 +86,32 @@ private static void setupDrivers(Application application) {
 		DriverFeature.addDriver("Macro", MacroFeature.getMacroDriverDecorator());
 
 		Job2dDriver loggerDriver = new LoggerDriver();
-		
+
 		DriverChangeTitleObserver driverObserver = new DriverChangeTitleObserver();
 		DriverFeature.getDriverManager().getChangePublisher().addSubscriber(driverObserver);
-		
+
 		DriverFeature.addDriver("Logger driver", loggerDriver);
-		
+
 		DrawPanelController drawerController = DrawerFeature.getDrawerController();
 		Job2dDriver driver = new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic");
 		DriverFeature.addDriver("Line Simulator", driver);
 		DriverFeature.getDriverManager().setCurrentDriver(driver);
+
+		DriverFeature.addDriver("Scale x2", new Jobs2dDriverTransformationDecorator(DriverFeature.getDriverManager().getCurrentDriver(), 2, 2));
```

Comment:
Transformation can work with different drivers but it is fixed. You should use driver change observer to transparently apply transformations to various drivers.